src/components/Header/header.scss

```diff
@@ -0,0 +1,198 @@
+@import "../../styles/main.scss";
+
+.header {
+   background: rgba(241, 235, 235, 0.5);
+   height: 60px;
+   width: 100%;
+   text-align: center;
+   z-index: 500;
+   position: sticky;
+   top: 0;
+   display: flex;
+   align-items: center;
+   justify-content: center;
+   padding: 2px 5px;
+   box-sizing: border-box;
+   @media (max-width: 426px) {
+    height: 40px;
+  }
+      &__menu {
+        width: 54px;
+        height: 47px;
+        margin: 12px, 0px, 0px, 8px;
+        cursor: pointer;
+        &:hover {
+        width: 60px;
+        height: 55px;
+        }
+        @media (max-width: 426px) {
+        width: 35px;
+        height: 30px;
+        }
+      }
+      &__right-block {
+        display: flex;
+        height: 30px;
+        align-items: center;
+        justify-content: center;
+        vertical-align: middle;
+        background: rgba(109, 107, 107, 0.15);
+        margin-right: 0px;
+        margin-left: auto;
+        border-radius: $buttonBorderRadius;
+        @media (max-width: 426px) {
+          display: grid;
+          align-items: center;
+          justify-content: right;
+          height: 40px;
+        }
+      }
+      @media (min-width: 992px) {
+        -ms-flex-flow: row nowrap;
+        flex-flow: row nowrap;
+        -ms-flex-pack: start;
+        justify-content: flex-start;
+      }
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  position: absolute;
+  left: 50%;
+  transform: translate(-50%, 0%);
+      &:active, &:focus, &:hover{
+        text-decoration: none;
+      }
+        &__textEvents {
+          font-family: Roboto;
```

Comment:
Merge all font properties into one, and create variable for it, because it uses not only in this place